packages/governance/test/unitTests/test-binaryballotCount.js

```diff
@@ -422,22 +432,3 @@ test('binary no quorum', async t => {
     reason: 'No quorum',
   });
 });
-
-test('binary too many positions', async t => {
```

Comment:
Rather than dropping this test, it would be better to change it to `test.fails`  or change `t.throws()` to `t.notThrows()`. The fact that it doesn't fail with 3 positions is worth testing.